File: java/client/src/main/java/glide/api/models/exceptions/ExecAbortException.java
/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
package glide.api.models.exceptions;

/** Exec aborted error: Errors that are thrown when an Atomic Batch (Transaction) is aborted. */
public class ExecAbortException extends GlideException {
    public ExecAbortException(String message) {
        super(message);
    }
}

File: csharp/sources/Valkey.Glide.InterOp/Errors/ExecAbortException.cs
namespace Valkey.Glide.InterOp.Errors;

/// <summary>
/// An error on Valkey service-side that is thrown when a transaction is aborted
/// </summary>
public sealed class ExecAbortException : GlideException
{
    public ExecAbortException() : base() { }

    public ExecAbortException(string message) : base(message) { }

    public ExecAbortException(string message, Exception innerException) : base(message, innerException) { }
}